doc/changes/devel/12324.bugfix.rst

```diff
@@ -0,0 +1 @@
+Add ``tol`` parameter to :meth:`mne.events_from_annotations`, by `Michiru Kaneda`_
```

Comment:
can you be more explicit for this sentence? it's written as an enhancement and not really a bug fix. Something like

Add ``tol`` parameter to :meth:`mne.events_from_annotations` so that ... when using `chunk_duration != None`, by `Michiru Kaneda`_

Reply:
Here is an update:
https://github.com/mne-tools/mne-python/pull/12324/commits/7dd73754cff3bc2fe304a9bf8fd8d4a321482638